tools/automator-main.sh

```diff
@@ -226,6 +231,15 @@ validate_opts() {
   if [ -z "${git_exclude:-}" ]; then
     git_exclude=""
   fi
+
+  if $fetch_tags; then
+    if [ -n "$merge_branch" ]; then
+      tag_branch="$(git describe --tags --abbrev=0 "$merge_branch")"
```

Comment:
Is this happening at the right place? At this point we didn't even clone the repo, I think the `git describe` is running against this (test-infra) repo, not the target one.

Reply:
I was doing the login in another part. I removed from here and add it into the merge function. The test passed using the parameter and without it (using my local repos), but please validate it too if possible.